config/permissions.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Twill permissions
+    |--------------------------------------------------------------------------
+    |
+    | This allows you to set up the level of management of the permissions.
+    |
+    | Three levels exists
+    | - role: this manages the access of modules based on the role of a user.
+    | - roleGroup: this manages the access of modules for a group of users.
+    | - roleGroupItem: this manages the permission of items within a module.
+    |
+     */
+
+    'level' => 'role',
+    'modules' => [], // List of the modules the application have to manage access
```

Comment:
@pboivin should this apply automatically to all if not set?

Reply:
Refreshing my memory on how roles are converted from the legacy system : [1](https://github.com/area17/twill/pull/1138/files#diff-039f5870c33f40d39f43a227f7e47a588b2a91c6d6459c85e0f44381fac8bb71R53) [2](https://github.com/area17/twill/pull/1138/files#diff-f6c89eeb9d2a9e7ae3333696a19a70d9591d5f700ca4c6389ed09648e9baaef2R148)

Seems like making all modules "permissionable" by default would also be non-breaking when upgrading from the default 2.x setup 👍️

Reply:
I think it makes sense to keep it "to be configured". So I will leave as is.